tests/wrapper/test_pagamentos.py

```diff
@@ -228,41 +230,22 @@ def test_criar_dados_com_email_invalido(self):
         Teste para verificar e montar os dados da transferência pix usando email como chave # noqa
         """
 
-        expected_json = {
-            "numeroRequisicao": "123",
-            "agenciaDebito": "345",
-            "contaCorrenteDebito": "678",
-            "digitoVerificadorContaCorrente": "X",
-            "tipoPagamento": 128,
-            "listaTransferencias": [
-                {
-                    "email": "",
-                    "cnpj": None,
-                    "cpf": None,
-                    "dddTelefone": None,
-                    "telefone": None,
-                    "data": "19042023",
-                    "valor": 11.0,
-                    "descricaoPagamento": "Pagamento",
-                    "formaIdentificacao": 2,
-                    "identificacaoAleatoria": None,
-                }
-            ],
-        }
-
-        response = PagamentoLoteBBWrapper()._criar_dados_transferencia_pix(
-            "123",
-            "345",
-            "678",
-            "X",
-            "19042023",
-            11,
-            "teste@...",
-            "Pagamento",
-            128,
-        )
-
-        self.assertEqual(expected_json, response)
+        with self.assertRaises(ValueError) as ctx:
+            PagamentoLoteBBWrapper()._criar_dados_transferencia_pix(
+                "123",
+                "345",
+                "678",
+                "X",
+                "19042023",
+                11,
+                "teste@...",
+                "Pagamento",
+                128,
```

Comment:
quando comentei me referi a esse tipo de kwarg

```suggestion
                n_requisicao="123",
                agencia="345",
                conta="678",
                dv_conta="X",
                data_transferencia="19042023",
                valor_transferencia=11,
                chave="teste@...",
                descricao="Pagamento",
                tipo_pagamento=128,
```